### PR TITLE
Include error information on superadmin responses

### LINKED ISSUE #6785: Add better failure info to Superadmin API
Have the API return more precise errors to be recovered by central. We can agree on the format.

### PR DESCRIPTION
**Pull request by javitonino**
Closes #6785

### CODE REVIEW

**Comment by javitonino** [comment]
CR please @juanignaciosl

File: app/controllers/superadmin/superadmin_controller.rb [hunk 1]
```diff
@@ -24,4 +26,11 @@ def authenticate
   def current_user
     super(CartoDB.extract_subdomain(request))
   end
+
+  private
+
+  def rescue_from_superadmin_error(error)
+    CartoDB::Logger.error(exception: error)
```
Comment (juanignaciosl) [inline app/controllers/superadmin/superadmin_controller.rb]: Is the request automatically logged? I'd add request information if it's not.

Comment (javitonino) [inline app/controllers/superadmin/superadmin_controller.rb]: Yes, request is automatically logged, at least URL, headers, ... I will double-check for request body, which could be very useful here.

Comment (javitonino) [inline app/controllers/superadmin/superadmin_controller.rb]: Yes, body is also logged automatically :smile:

Comment (juanignaciosl) [inline app/controllers/superadmin/superadmin_controller.rb]: Good to know what their _convention over parameter passing_ is :D

**Comment by javitonino** [comment]
Just to confirm, @juanignaciosl, is this ready?

**Comment by juanignaciosl** [comment]
Yes, my bad, sorry: I wasn't explicit. :shipit: !

**Pull request merged**